CompuCell3D/CMakeLists.txt

```diff
@@ -22,7 +22,16 @@ ENDIF (APPLE)
 
 set(CMAKE_CXX_STANDARD 11)
 
-FIND_PACKAGE (OpenMP)
+
+FIND_PACKAGE(OpenMP)
+IF(OPENMP_FOUND)
+message ("openmp c flags " "${OpenMP_C_FLAGS}")
+message ("openmp cxx flags " "${OpenMP_CXX_FLAGS}")
+SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS} ")
+#SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS} -std=c++11 -stdlib=libc++")
```

Comment:
Fails with gcc. See fix in https://github.com/CompuCell3D/CompuCell3D/pull/509.

Reply:
I am sure it does. I remember breaking things to get it working on OSX. Forgot to clean up :)